tls/main.go

```diff
@@ -217,6 +219,13 @@ func init() {
 			EnvVars:     []string{"SERVICE_AUTH"},
 			Destination: &tlsConfigValues.Auth,
 		},
+		&cli.StringFlag{
```

Comment:
In order to set the new metrics configuration parameter, it can be done adding a new `BoolFlag` to set it. See the `config-file` or the `logger` flags as examples.